dashboard/app/models/levels/dancelab.rb

```diff
@@ -58,6 +59,10 @@ def uses_google_blockly?
     true
   end
 
+  def uses_lab2?
+    uses_lab2
```

Comment:
Nit: force this to return a boolean for levels without this attribute?

Reply:
Hmm, shouldn't `uses_lab2?` return `false` if `uses_lab2` is not present? Unless i'm misunderstanding something - or do you mean for levels other than dance?

Reply:
Oh snap nvm, didn't know we had this little helper for serialized attributes that already does this!

https://github.com/code-dot-org/code-dot-org/blob/abf0fdbfc40344e85c5f1f6e58073f89adbf2c34/dashboard/app/models/concerns/serialized_properties.rb#L63

Reply:
Actually which makes me think you might not need this method at all?

Reply:
Oh interesting, good find! Didn't realize we had this helper. Yeah, seems like we don't need the function at all then - I tried calling `uses_lab2?` on a Dance level (has the function) and a different level (without the function) and got `false` in both cases.